android/app/src/main/java/com/sample/MainActivity.kt

```diff
@@ -22,11 +25,17 @@ class MainActivity : ReactActivity() {
     @Inject
     lateinit var manifestProvider: ManifestProvider
 
+    @Inject
+    lateinit var bundleNameProvider: ReactBundleNameProvider
+
+    @Inject
+    lateinit var testReactNativeVersion: TestAppReactNativeHost
```

Comment:
`ReactActivity` has a getter for `ReactNativeHost`. Can't we just add a getter that casts it to the correct type?